src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -149,10 +143,11 @@ class CrudField {
         subfield(name, rowNumber = false) {
             let subfield = new CrudField(this.name);
             subfield.name = name;
+
             if(!rowNumber) {
                 subfield.isSubfield = true;
-                subfield.subfieldHolder = this.name;
```

Comment:
Shouldn't we ALSO keep `subfieldHolder` for now... even if undocumented, to make this a non-breaking change? What if someone is already using it?

Reply:
@tabacitu I added it back 👌 I think it is super unlikely that someone is using it, because it is used internally only on subfields, anyway, no problem on keeping it.

Reply:
Excellent. Love the `deprecated` comment.